CSS/index.css

```diff
@@ -250,6 +245,79 @@ a{
     font-size: 20px;
 }
 
+/* Section 3 Elements--------------------------------- */
+
+.section-3{
+    width: 100%;
+    height: 400px;
+    background-color: rgba(235, 235, 235, 0.9);
+    align-items: center;
+    justify-content: flex-start;
+    position: relative;
+    gap: 8em;
+}
+
+/* Entire quote block */
+.blockquote{
+    max-width: 70em;
+    max-height: 12em;
+    margin-left: 80px;
+    margin-right: 80px;
+    margin-top: 10px;
+    margin-bottom: 90px;
+    padding: 25px;
+    background: #F8FFFE;
+    border: 1px solid #7CCDDA;
+    color: #253D41;
+}
+
+ /* The border width on the left side of the quote block */
+.blockquote-bordered{
+    border-left-width: 12px;
+}
+
+/* Open quotes symbole before the quote */
+.quote-mark::before{
```

Comment:
pseudo-classes were used to create an open quotation mark ( " ) as required by #5